source/fearless/sharing.d

```diff
@@ -51,6 +51,8 @@ static if (is(typeof({import automem.ref_counted;}))) {
         else
             return RefCounted!(Exclusive!T, Allocator)(allocator, args);
     }
+}else{
```

Comment:
I don't think this is right either - most clients of this library won't have automem and nobody wants a spurious compiler message.